databricks-sdk-java/src/main/java/com/databricks/sdk/core/ApiClient.java

```diff
@@ -29,55 +30,119 @@
  * guessing
  */
 public class ApiClient {
+  public static class Builder {
+    private Timer timer;
+    private Function<Void, Map<String, String>> authenticateFunc;
+    private Function<Void, String> getHostFunc;
+    private Function<Void, String> getAuthTypeFunc;
+    private Integer debugTruncateBytes;
+    private HttpClient httpClient;
+    private String accountId;
+    private RetryStrategyPicker retryStrategyPicker;
+    private boolean isDebugHeaders;
+
+    public Builder withDatabricksConfig(DatabricksConfig config) {
```

Comment:
Actually, this could even be private, since it is only used in the default constructor.